### ISSUE TITLE
"Authorization required, but no authorization protocol specified" after installing something

### ISSUE THREAD

**Issue by taukakao** [Vanilla-OS/vso-image#15]
### Issue Description

This is because of the apt post hook https://github.com/Vanilla-OS/vso-image/blob/main/includes.container/usr/share/vso/hooks/apt-post

The system seems to be trying to create a new dbus instance which kills the old one. Not sure about the details here yet.

The user has to reboot to start graphical programs again. 

### Steps to Reproduce

- run `sudo su -c 'XDG_RUNTIME_DIR=/run/user/$UID; DBUS_SESSION_BUS_ADDRESS='unix:path=/run/user/$UID/bus'; host-shell ls' $SUDO_USER`
- run any graphical app like gedit

### On what version of Vanilla OS this happens?

Unreleased

**Comment by taukakao** [comment]
Also, after running the above command systemd outputs:
```
session-3.scope: Deactivated successfully.
user@1001.service: Deactivated successfully.
run-user-1001.mount: Deactivated successfully.
user-runtime-dir@1001.service: Deactivated successfully.
```

**Comment by taukakao** [comment]
Maybe implementing this with a systemd service that runs as the actual user would be easier

**Comment by taukakao** [comment]
Actually just running 
`sudo su -c ls your_username`
can replicate this issue.

**Comment by taukakao** [comment]
Oh, I found something that works:

```
sudo su -c "su -c 'XDG_RUNTIME_DIR=/run/user/1001; DBUS_SESSION_BUS_ADDRESS='unix:path=/run/user/1001/bus'; host-shell vso export --app firefox' your_username"
```

replace 1001 with your uid and gid and your_username with your username

**Comment by taukakao** [comment]
Will be closed by #16
